docs/interfaces.md

```diff
@@ -45,6 +45,14 @@ apps interfering with one another.
 Usage: reserved
 Auto-Connect: yes
 
+### pulseaudio
+
+Can access the PulseAudio sound server. Allows for sound playback in games and
+media application.
+
+Usage: common
+Auto-Connect: yes
```

Comment:
Just for pondering, if we wanted to split the recording from the playback interface, how could we do that?